ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/fabric_numbering.schema.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  fabric_numbering:
+    description: Automatic assignment of numbers for settings like Node ID.
+    type: dict
+    keys:
+      id:
+        type: dict
+        description: Assignement of Node ID.
+        keys:
+          algorithm:
+            type: str
+            valid_values:
+              - pool_manager
+            description: |-
+              If set, IDs will be automatically assigned using the configured algorithm unless statically configured under the node settings.
+
+              - "pool_manager" will activate the pool manager for ID pools.
+
+                The pools are dynamically built and matched on the following device data:
+                - `fabric_name`
+                - `dc_name`
+                - `pod_name`
+                - `type`
+
+                Note: This means changing any of these fields may renumber the devices!
+
+                Each pool will assign the first available ID starting from 1.
+
+                Stale entries will be reclaimed from each pool automatically after every run.
```

Comment:
It may be overkill but also may be useful to offer an example for how to re-use stale entries. Something to the effect of `i.e. If you are replacing a particular node, node A, that is using id:1 with another host, node B that you wish to use id:1, first you will need to remove node A from the inventory file, run the eos_designs playbook successfully, then add node B to the inventory file (in the same location node A was), and then re-run the playbook a second time successfully.`

Reply:
I will try to move some of this doc to the markdown instead of the table. That will make it easier to be more elaborate.

Reply:
I will update the docs

Reply:
updated.